### DISCUSSION TITLE
Selenium 4 compatibility

### DISCUSSION THREAD

**Discussion by dfsmarsi** [appium/appium#17139] [General]
Hello everyone, do we currently have compatibility with Selenium 4, or only up to version 3.141?

**Comment by RSM-SBA** [comment]
@dfsmarsi 
I use Selenium 4.1.3 java-client
with appium 2 beta
and appium java-client 8.1

**Reply by Auto81** [reply]
@StephanB75 are you running tests on native apps or doing browser testing? 

I've got a a grid 4 hub running, with a relay node registered out to an appium 2 server. Driver starts the app up then immediately crashes out with `org.openqa.selenium.SessionNotCreatedException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.`

Have tried your versions, and also

java-selenium@4.4.0
appium java-client@8.2.0
appium 2

**Reply by RSM-SBA** [reply]
I do native app testing but not over a grid hub.
